### ISSUE TITLE
check:dead_controllers reports single action controller as error

### ISSUE THREAD

**Issue by PatrickSJ** [imanghafoori1/laravel-microscope#248]
A controller using `__invoke` is reported as an error when running `check:dead_controllers`.  This is happening for all routes to a controller using `__invoke`.  The routes file has defined all the routes with errors in the following style:

```
Route::post('some-route', SomeController::class)->name('some-route-name');
```

```
class SomeController extends Controller
{
    public function __invoke(Request $request) {
        
    }
}
```

Laravel Framework v9.48.0
PHP 8.1.7
Laravel-Microscope v1.0.281

**Comment by Mohammadreza-73** [comment]
I have same issue.

**Comment by imanghafoori1** [comment]
fixed as of version 1.0.347
